core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -744,9 +744,16 @@ public void sendTo(Address destination, ReplicableCommand rpcCommand, DeliverOrd
          }
          return;
       }
-      dispatcher.sendMessage(
+      CommandAwareRpcDispatcher rpcDispatcher = dispatcher;
+      if (rpcDispatcher == null) {
+         if (trace) {
+            log.trace("Don't send message. Transport is stopped.");
```

Comment:
Maybe we should have a private method that does the null check and logs the message?  This way we don't have to duplicate the code check in multiple places.